MarkdownApp/Shared/ViewModels/Interfaces/IMarkDownViewModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using MarkdownApp.Shared.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MarkdownApp.Shared.ViewModels.Interfaces
+{
+    public interface IMarkDownViewModel
```

Comment:
why do u need this interface?

Reply:
no, the idea was to create a repository or service rather than "viewModels" to call the controller, but end it up having a useless interfaces and classes.